pkg/render/common/components/components.go

```diff
@@ -118,10 +125,11 @@ func ApplyDeploymentOverrides(d *appsv1.Deployment, overrides components.Replica
 
 	// Pull out the data we'll override from the DaemonSet.
 	r := &replicatedPodResource{
-		labels:          d.Labels,
-		annotations:     d.Annotations,
-		minReadySeconds: &d.Spec.MinReadySeconds,
-		podTemplateSpec: &d.Spec.Template,
+		labels:             d.Labels,
+		annotations:        d.Annotations,
+		minReadySeconds:    &d.Spec.MinReadySeconds,
+		podTemplateSpec:    &d.Spec.Template,
+		deploymentStrategy: &d.Spec.Strategy,
```

Comment:
I think we should add MinReadySeconds in `ApplyDaemonSetOverrides` as well: https://github.com/tigera/operator/pull/2303/files#diff-39f0bed05108326edaac1c8f66574d43d5ed843ff91ff6670e813223dac9b3a1R106

DeploymentStrategy doesn't apply there, but if someone wants to extend this to allow configuration of a DaemonSet minReadySeconds, would be nice if it was just a change to the corresponding ReplicatedPodResourceOverrides rather than also needing to make a change here.

Even though we're not using them now, if they get added to a DaemonSet in the future it will "just work" - we've already implemented `Get

Reply:
I think it's already there (was before my change).  Maybe just the formatting changed here to make it look like I added it?